src/FormHandler/ThreadFormHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace BiffBangPow\MessageBoard\FormHandler;
+
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use BiffBangPow\MessageBoard\Model\Thread;
+
+class ThreadFormHandler
```

Comment:
it would be good to have a unit test for this class. Might be worth pairing with either me, Will or Chris for this?